ClientApp/viking-vault/src/UserDashboard/ExchangeForm.css

```diff
@@ -88,15 +85,14 @@
     color: rgb(29, 84, 95);
     font-size: 120%;
     font-family: Helvetica;
-    font-weight: 750;
 }
 
 .white-box-background {
```

Comment:
this `.white-box-background` is more than a white box with a background though, you also set the width and margin etc.. think of a better way to name CSS classes or split them into 2 classes, one for a specific kind of box styling (lets say one with a with background with a box shadow) and another one where you add things such as `margin` and all that positioning to be a more specific class, and then this becomes `.classOne.classTwo`